src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -3,6 +3,7 @@
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Diagnostics;
+using System.Linq;
```

Comment:
Do we need this `using`?

Reply:
Used for `Parameters.Any()`.

Reply:
My bad. I thought `ImmutableArrayExtensions` was in one of our namespaces. Thanks